components/item/AwesomeSwiper.vue

```diff
@@ -59,6 +60,7 @@
           slidesPerView: 'auto',
           spaceBetween: 40,
           centeredSlides: true,
+          slideToClickedSlide: true,
```

Comment:
Is this a swiper option? Or is it used?

Reply:
yes it's a built-in option which achieves this requirement: “When a resource is not the current focus image, clicking it brings it to the focus”

Reply:
Thanks for clarifying!